client/index.html

```diff
@@ -0,0 +1,171 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+	<title>Chat prototype</title>
+	<script src="js/jquery-3.2.1.min.js"></script>
+	<script src="js/dataManager.js"></script>
+    <script src="https://unpkg.com/react@15/dist/react.js"></script>
+    <script src="https://unpkg.com/react-dom@15/dist/react-dom.js"></script>
+    <script src="js/react/browser.min.js"></script>
+</head>
+<body>
+	<h1>
+		SjChat
+	</h1>
+	<script>
+		var apiAddress = window.location.protocol +'//'+ window.location.hostname +':8080';
+		var dataManager = new DataManager();
+
+		$.ajaxSetup({
+			async: false
+ 		});
+
+		// Add chats
+		$.get(apiAddress+"/chat", function(data){
+			var chats = data;
+
+			for (let chat of chats){
+				dataManager.putChat(chat);
+
+				for (let user of chat.users){
+					dataManager.putUser(user);
+				}
+			}
+		});
+
+		// Add messages to chats
+		for (var chatId in dataManager.chats){
+			$.get(apiAddress+"/chat/"+chatId+"/message", function(data){
+				var messages = data;
+
+				for (let message of messages){
+					dataManager.putMessage(chatId, message);
+				}
+			});
+		}
+
+	</script>
+
+	<div style="width: 30%; background-color: #FF55AA; float: left">
```

Comment:
For now it's okay to inline the css. But in the future we should not use any inlined css at all. I created an issue for this #77.